source/agora/common/TransactionPool.d

```diff
@@ -56,6 +56,8 @@ public class TransactionPool
     /// SQLite db instance
     private Database db;
 
+    // map for Input objects of each transaction.
+    private Input[Hash] input_map;
```

Comment:
The `Input` itself doesn't seem to be used. So this could be a `Set!Hash` instead. There is a Set implementation in `agora.common.Set`